pkgs/by-name/he/hexxy/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+buildGoModule {
+  pname = "hexxy";
+  version = "0-unstable-2024-06-07";
+  src = fetchFromGitHub {
+    owner = "sweetbbak";
+    repo = "hexxy";
+    # upstream does not publish releases, i.e., there are no tags
+    rev = "30e0aa5549bbafeb8204fe34b0d37019f9acc975";
+    hash = "sha256-KBgxZD95UT7i/eYeKLm0LVLliKgK/KiJYXVY9zzwbvk=";
+  };
+
+  vendorHash = "sha256-qkBpSVLWZPRgS9bqOVUWHpyj8z/nheQJON3vJOwPUj4=";
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  passthru.updateScript = unstableGitUpdater { hardcodeZeroVersion = true; };
```

Comment:
I just find that `unstableGitUpdater` cannot handle `vendorHash`.

`nix-update-script` supports `vendorHash`.  However, it does not work on my machine with this:

```nix
passthru.updateScript = nix-update-script { extraArgs = [ "--version" "unstable" ]; };
```

Reply:
If it does not work on your machine, why do you mention it?

Reply:
This should work:
```nix
passthru.updateScript = nix-update-script { extraArgs = [ "--version" "branch" ]; };
```

---
> If it does not work on your machine, why do you mention it?

Not sure what you mean by "mention it".

I [mention][1] `unstableGitUpdater` because the author originally use `gitUpdater` which does not work because this program lacks tags.

Then, I notice that `vendorHash` is used.  So I mention `nix-update-script` as a potential replacement because I know it supports unstable version update and `vendorHash` update.

However, my local test with `nix-update-script` does not work, which I mention in the hope of others chiming in to help.

[1]: https://github.com/NixOS/nixpkgs/pull/318012#discussion_r1631891656

Reply:
OK fair enough. I was not understanding your comment, mentioning another potential failing solution that would not help. But OK never mind.